src/main/java/functions/Save.java

```diff
@@ -0,0 +1,64 @@
+package functions;
+
+import functions.TaskList;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.StandardCopyOption;
+
+public class Save {
+    TaskList taskList;
+    private String saveFilePath;
+    private static final String tempFilePath = "temp.txt";
```

Comment:
Maybe this constant could be in all-caps w/ underscores to seperate words?
```suggestion
    private static final String TEMP_FILE_PATH= "temp.txt";
```